### ISSUE TITLE
Questions in gender_role_items to leave unused?

### ISSUE THREAD

**Issue by fsolt** [fsolt/dcpo_gender_roles#1]
Not sure we should use a question?  Talk about it here

**Comment by byngdeuk** [comment]
There are several items that I have an issue with.
Wvs a075, Wvs a092, Evs a075, Evs a092, EVS d065, LB A503101C, LB A505226, LB A505306
LB A505307, AmB cp20, ISSP_1994 v14, CEE v140, CEE v143, CEE v168, ELL_92-3 v16
ELL_92-3 v26, ELL_92-3 v36, ELL_92-3 v46, ELL_92-3 v57, ELL_92-3 v67, Estonia_95 v14
Estonia_95 v24, Latvia_1995 v14, Latvia_1995 v24

Moreover, I wonder whether the below pairs of questions are meaningful when they are compared?
ISSP_1988 v33 & ISSP_1988 v34
NSSS_8990 LONEMO & NSSS_8990 LONEFA

**Comment by fsolt** [comment]
@byngdeuk Looking first at the single-mother (ISSP_1988 v33 & NSSS_8990 LONEMO) and single-father  (ISSP_1988 v34 & NSSS_8990 LONEFA) questions, my take would be that (1) DCPO doesn't provide for looking at _differences_ in responses to questions (if that's what you mean by comparing the pairs), and (2) I don't think we want to that anyway: each of these questions provides some information on the extent to which people adhere to traditional-family gender roles rather than more egalitarian ones.  Sure, these questions tap other things too (like common sense--two people are in most cases going to be better at accomplishing things together than one person is alone), but single parents--regardless of whether they are mothers or fathers--by taking on both breadwinning and nurturing duties, are necessarily violating traditional gender roles and embodying egalitarian ones.  Well, that's my thought, anyway.  What are your reactions to that, @byngdeuk and @lallemang?

**Comment by fsolt** [comment]
@byngdeuk 
I agree we should exclude WVS/EVS a075 and a092, LB A503101C,  AmB cp20, CEE v140, CEE v143, ELL_92-3 v16, ELL_92-3 v26, ELL_92-3 v36, ELL_92-3 v46, ELL_92-3 v57, ELL_92-3 v67, Estonia_95 v14, Estonia_95 v24, Latvia_1995 v14, and Latvia_1995 v24, which all have to do with belonging to/volunteering for/working for women's groups.  Totally orthogonal to what we want.

I (more reluctantly?) also agree we should exclude EVS d065, "Men are less able to handle emotions in relationships than women".  I guess we wouldn't be super confident that this question taps the traditionalist/egalitarian dimension very well, if at all.  @lallemang?

I agree we should exclude LB A505226, "Conflict between men and women: In all countries there are differences or even conflicts between different social groups. In your opinion, how much conflict is there between...?. Men and women."  Not enough face validity--differences/conflict could reflect both ends of our scale.

OTOH, LB A505306 and LBA505307, "Flattering or offensive: How do you consider the following things that can happen to a woman from a person who is not her partner?  Whispered compliment in the ear"/"Being rubbed against on purpose" tap recognition of sexual harassment—a mechanism by which men defend privileged positions against gains by women—and responses should be useful information.  I see the point that they might well be ___very___ noisy, but my view would be that we should at least try these items before giving up on them.  Further thoughts, @byngdeuk and @lallemang?

I think I favor keeping ISSP_1994 v14, "Family life often suffers because men concentrate too much on their work."  In the traditional view, there's no way that this could be true; while from a more egalitarian viewpoint, fathers should be doing less breadwinning and spending more time nurturing.  Further thoughts, @byngdeuk and @lallemang?

I see the point of excluding CEE v168, "People associate democracy with diverse meanings such as those on this card. For each of them, please tell me whether, for you, it has a lot, something, not much, or nothing to do with democracy, Equal rights for women.": we care about gender equality, but not democracy, right?  But the fact that the question has democracy in it doesn't render it useless for our purposes.  As long as egalitarians are more likely to give (high) responses, and traditionalists are more likely to give (low) responses, responses to this question will tell us something about how many egalitarians and traditionalists there are in the country.  We should keep this one.

While I'm looking at these, I don't think we should use LB A505221A, "Employment discrimination by sex: Would you say that in equal conditions of grades, or titles a person is more likely, the same probability, or less likely that would be promoted or accepted in a job?. A woman compared to a man", since it is a positive rather than normative question.  In other words, a traditionalist and an egalitarian can very well give the same response to this question—sure, one will be pleased and the other unhappy, but we won't have that information, and so we won't be able to tell them apart.  Similar story for ISSP_1985 v34, "Would you say that opportunities for university education <USA: college education> are, in general, better or worse, for women or for men?"  There may be others like this, too.

**Comment by byngdeuk** [comment]
**First about Single Mother and Father questions,**
I agree with that those questions are related to gender roles because they compare between single parent and married couple. For example, if people think that women (or father) alone cannot bring up their child as normal (married couple), it gives signals that the people have traditional gender role perspectives. Thus, I also think that those questions are related to our purpose.

**Second about other questions,**

ISSP_1994 v14 "Family life often suffers because men concentrate too much on their work." 
-> I agree. If people have egalitarian view, they think that family will suffer when men concentrate too much on their work. House stuffs, raising kids, earning money, etc. should be equally done by men and women based on egalitarian view. 

CEE v168 "about democracy"
->I understand why democracy, equal rights, and egalitarian view are connected. I agree with keeping this question.

LB A505306 and LBA505307 "Flattering or offensive"
-> I understand why those two questions can be useful. If people think flattering or whispering from person who is not a partner, the people tend to have more egalitarian gender role.

 LB A505221A and ISSP_1985 v34 "equal opportunity"
-> I understand the logics. Those questions make respondents to talk about reality, not about opinion. I agree with dropping those kinds of questions.

**Comment by fsolt** [comment]
@byngdeuk 
Just to be sure we're on the same page re LB A505306 and LBA505307, here are the original questions in the English survey:

<img width="499" alt="Screen Shot 2020-02-24 at 12 39 02 PM" src="https://user-images.githubusercontent.com/6536645/75180867-aa7b9b80-5702-11ea-8bea-acb437d5b1dd.png">

Recognizing that the woman should be offended is the egalitarian position.  "Oh, you should be flattered!" is the traditional view.

**Comment by byngdeuk** [comment]
@fsolt 
Yes! offensive = egalitarian position / flattering = traditional view.

**Comment by lallemang** [comment]
Sorry again for the delay in getting back to you two about these questions. Here are my thoughts:

**Single mother/father:**
I agree that these questions are tapping into opinions on gender roles, given the gendered nature of divvying up parenting duties within presumably married couples. Therefore, these questions are potentially useful to us. 
**Women’s groups:**
I agree that these questions are not pertinent to traditional/egalitarian opinions on gender roles. So, I am also fine with excluding these questions.
**Men handling emotions:**
I am also a little stuck on this one, and I understand justifications for keeping or discarding this question. I think the confusion comes down to how we would except someone with egalitarian views on gender roles to respond. I’m assuming we all agree that a traditionalist would respond as agreeing that men are less able to handle emotions in relationships with women. This is because they would believe that men shouldn’t have to handle emotions in relationships – Emotional burdens ought to fall on women. Women are emotional, and men are logical – These are the proper roles. But it is less obvious to me how an egalitarian might respond. On the one hand, they may also agree with the statement because they believe that men are socially conditioned to relegate dealing with emotions to women. Men do not have to be bad at handling emotions, but they are because of pervasive norms regarding gender roles. On the other hand, an egalitarian may be aware of this social conditioning, and that’s precisely why they would respond disagreeing with the statement. From this perspective, men are just as able to handle emotions as women, but they do not do so in practice because of gendered norms. How we expect an egalitarian to respond is important because if we expect the first perspective to be more likely, then both an egalitarian and a traditionalist would agree with the statement in the survey question, and we wouldn’t be able to differentiate which opinion on gender roles the answer is reflecting. However, if we think the second egalitarian perspective I proposed is more likely, then I think we could keep this question and find it useful.
**Conflict between men and women:**
I agree that there is not enough face validity to this question.
**Flattering or offensive, regarding sexual harassment:**
I think we should definitely keep these questions because they tap into gendered expectations women face to submit to sexual advances and to be thankful for them as signals of women’s worth – validating and strengthening a gender hierarchy that privileges men. Opinions on this question will certainly reveal traditionalist versus egalitarian views.
**Family life suffers because men concentrate too much on their work:**
I agree that this is worth keeping. Traditionalists won’t agree with this, and egalitarians are more likely to agree because they believe that fathers should equally contribute to all family duties and not solely focus on breadwinning. 
**Democracy and gender equality:**
I agree with keeping this question. If egalitarians are more likely to give high responses and traditionalists low responses, then I agree with Fred that this question is useful for our purposes, even though it’s about democracy. 
**Employment discrimination and opportunities for education questions:**
While these are interesting questions that are certainly related to gender equality, I agree with Fred that we would not be able to discern traditionalist answers from egalitarian answers. This one is more cut-and-dry for me than the question about men being able to handle emotions. I don’t see a way that we could differentiate answers for these questions, so I agree with not using them.

**Comment by fsolt** [comment]
So it looks to me like we've reached consensus on everything except EVS d065, "Men are less able to handle emotions in relationships than women."  My read on this question is very similar to yours, @lallemang, I think.  On one end of the scale, traditionalists will definitely agree.  On the other end, egalitarians may agree, on the positive basis that toxic masculinity strips (many/most) men of the full emotional range required of healthy relationships (part of the message behind 'toxic masculinity hurts everyone'), but they may also disagree on the normative (and, to some extent, positive) basis that it should not and doesn't have to be this way.  My original take was that the former would outweigh the latter and whatever signal available in the data would be overwhelmed by noise.  Maybe the better view, though, is that we should just try it and see.

**Comment by fsolt** [comment]
I'm interested in hearing your thoughts re the following questions on "the women's movement":

WVS | e069_15 | For each one, could you tell me how much confidence you have in them? The women's movement
WVS | e108 | For each of the following movements, which I read out, can you tell me whether you approve or disapprove of this movement? Women's movement
EVS | e108 | For each of the following movements, which I read out, can you tell me whether you approve or disapprove of this movement? Women's movement
EB | MVANW | There are a number of groups and movements seeking the support of the public. For each of the movements, can you tell me whether you approve (strongly or somewhat) or disapprove (somewhat or strongly)? The women's movement

First, WVS e069_15 asks about "confidence in", which seems to me to have all kinds of problems for our purposes.  Maybe you lack confidence because (1) you're traditionalist and disagree, (2) you're egalitarian but you live in a traditionalist society and the movement doesn't stand much of a chance, (3) there are particularities of the movement in your country you don't like, (4) you don't like 'movements' in general . . .  Anyway, this one seems bad to me, just too multidimensional.

The rest are "approval" questions seem better, but here too I think there's the anchoring problem suggested by (3) above—people aren't responding to a single, monolithic, global women's movement, they are responding to a movement that exists in their country at the time they are surveyed.  Moreover, a disapproval doesn't convey directionality: one can disapprove because the movement has gone "too far" and because it hasn't gone "far enough".  So I don't think I like these either.

What are your thoughts?

**Comment by byngdeuk** [comment]
Sorry for the late response,
The below is my though.

About WVS | e069_15 | For each one, could you tell me how much confidence you have in them? 
I also think this question is hard to be included in DCPO because of "confidence" in the question. 

About other questions (WVS | e108, EVS | e108 , and EB | MVANW,
I agree with the thing that it is based on the existing women's movement in respondents' societies.
Based on South Korea's case, there is far extreme women's movement on my perspective.
Thus, even though I support "egalitarian" view, I will say "disapprove" for those questions.

**Comment by lallemang** [comment]
Hi Fred and Byung-Deuk,

I've been working on double-checking the survey items Byung-Deuk has coded so far, and below is a list of questions that I'm not sure we should keep. I have not gotten to the other surveys that Byung-Deuk has coded yet, so this list only includes WVS questions. I've listed the survey item, the text of the question, and a brief explanation of my hesitancy to include it. Please let me know your thoughts or if you have any confusion about my explanations.

**WVS d057 _“Being a housewife is just as fulfilling as working for pay”_:** This doesn’t really tap into the perspective of the respondent regarding whether women should  or should not be housewives vs. part of the workforce. You could have the egalitarian view that women should be allowed to fully participate in the workforce AND also believe that some women find being a housewife just as fulfilling.

**WVS d062 _“A job is alright but what most women really want is a home and children”_:** Again, I’m not sure that this is tapping into public opinion about women’s proper roles in society. I can fully believe in equal access and opportunity for women but believe that they also care more about home and family than their jobs.

**WVS d063 _“Having a job is the best way for a woman to be an independent person”_:** Someone with a traditionalist view might agree with this statement, but more importantly, they believe that women shouldn’t be independent people – Hence, this question isn’t tapping into what we want to measure.

**WVS d066 _“If a woman earns more money than her husband, it's almost certain to cause problems”_:** A traditionalist and an egalitarian respondent might very well both agree on this – The difference is the egalitarian will be pissed and the traditionalist will not.

**WVS d067 _“In your opinion, how important are each of the following traits in a woman? Wearing a veil in public places”_:** This is not a question about gender roles as much as it’s a question about cultural norms and practices.

**WVS d068 _“In your opinion, how important are each of the following traits in a woman? Being a good mother”_:** This is picky of me, but the word “good” is what throws me off with this question. An egalitarian respondent won’t believe that women inherently need to be mothers, but they could believe it’s important for women who are mothers to be “good” at it. 

**WVS d069 _“In your opinion, how important are each of the following traits in a woman?. Being a good wife”_:** Again, the word “good” throws me off. Using myself as a benchmark for an egalitarian respondent, I don’t think women inherently need to be wives, but I believe it’s important that those who select into marriage are “good” to their spouses. So, I might “agree” with this statement, even though I’m not a traditionalist.

**WVS d070 _“In your opinion, how important are each of the following traits in a woman?. Being religious”_:** I don’t think we should blur lines between traditionalist views on gender roles and more conservative religious views on gender roles (although they often overlap). A respondent could have traditionalist views on women’s roles in society that are not grounded in any religion; so I don’t think this question taps into the traditionalist-egalitarian spectrum like we want it to.

**WVS d074 _“In your opinion, how important are each of the following traits in a woman?. Protect her family”_:** I’m not sure how “protecting” the family taps into egalitarian vs. traditional gender roles. On the one hand, the traditionalist might believe that protection is a man’s duty, not a woman’s. On the other hand, the traditionalist might also believe that children are primarily under their mother’s care (i.e. her protection). 

**WVS d075 _“In your opinion, how important are each of the following traits in a woman? Sociability”_:** I’m not sure this differentiates between egalitarians and traditionalists. Both types of respondents could find it important for women to be sociable – the egalitarian because they might just prefer that all people be pleasant to be around, and the traditionalist because they believe it’s extra important for women to be “sweet”, “nice”, “agreeable”, etc.

**WVS d076 _“It is acceptable for a man to have more than one wife”_:** Again, I don’t think we should blur lines between traditionalist gender roles and other opinions that bleed into religious/cultural practices. 

**WVS e069_15 _“For each one, could you tell me how much confidence you have in them? The women's movement”_:** This question doesn’t address gender roles. An egalitarian respondent could be disappointed by the achievements of the women’s movement, and therefore not have much faith in it, but still have egalitarian views toward women’s roles in society.

**WVS e233 _“Please tell me for each of the following things how essential you think it is as a characteristic of democracy. Women have the same rights as men”_:** This is a question about what defines democracy, not about “proper” gender roles in society.

**WVS f164 _“Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, would you please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that is presented? Islam requires woman to dress modestly but does not require cover face with veil”_:** Again, I don’t think we’re trying to measure views toward traditional religious practices.

**WVS f165 _“It is a violation of Islam for male and female university students to attend classes together”_:** This question measures interpretations of Islam, not “proper” gender roles in society.

**WVS f166 _“By requiring that a man treat all of his wives equally, the true intent of Islam is to prohibit a man from taking more than one wife”_:** Again, this question measures interpretations of Islam, not gender roles.

**WVS f174 _“Islam requires that a country with a majority of Muslims be governed by men of Islamic learning”_:** Again, this question measures interpretations of Islam, not gender roles.

**WVS f199 (scale of never justifiable to always justifiable) _“For a man to beat his wife”_:** I flip-flopped on this question. On the one hand, I think egalitarians will consistently report that it’s never justifiable for a man to beat his wife, while traditionalists are more likely to waver on this statement. On the other hand, I think plenty of traditionalists are not pro-domestic violence – they don’t view women as equal to men, but they don’t condone physical violence against women either. In fact, I think one common traditionalist take on gender roles is that women are inherently weak/passive/vulnerable and in need of extra protection relative to men. So from this perspective, violence against women is extra unacceptable relative to violence against men.

**Comment by fsolt** [comment]
Thanks again for flagging these, Lindsey: these questions definitely aren't ideal.  As we discussed in the weekly meeting, [the DCPO model](https://github.com/fsolt/dcpo_article/blob/master/paper/dcpo_article.pdf) has parameters to model the ways in which questions fall short of the purpose to which we will put them.  Most importantly, DCPO estimates each question's _difficulty_—in this case, how committed to gender equality a respondent would have to be to answer positively—and its _dispersion_—how likely is it that egalitarians and traditionalists will provide similar answers.  So we have some wiggle room for imperfection.  So long as on average egalitarians and traditionalists will answer differently, the question will be useful to us.

Questions I tentatively think are okay to include:

**WVS d057 “Being a housewife is just as fulfilling as working for pay”** I see your point.  Still, I expect that traditionalists will generally strongly agree, comparing housewives to their breadwinner husbands and seeing each satisfied playing their assigned role, many egalitarians will disagree, comparing the life of a housewife unfavorably to that of a working woman, on average the responses should be different across the traditionalist-egalitarian spectrum, and we should get useful information out of this.  That some egalitarians will say, 'oh, wait, I don't want to demean the important and too often gendered labor of housework and childrearing' and agree will only bump up the question's dispersion.

**WVS d062 “A job is alright but what most women really want is a home and children”**  Yep, but, I'd bet the implication that therefore there's no need to give women equal opportunities in the workforce is going to raise hackles among many egalitarians and push their average agreement down relative to traditionalists, giving us some useful information.

**WVS d063 “Having a job is the best way for a woman to be an independent person”**  Possibly.  My suspicion is that traditionalists will simply recoil from the idea of women having jobs and disagree that this is the "best way" to anything good—public opinion research is pretty solid that people respond to the question they want to answer more than the exact question asked.  If you're right, it'll blow up the question's dispersion, and we'll see it.

**WVS d066 “If a woman earns more money than her husband, it's almost certain to cause problems”**  Yes, I see the potential for the situation you described, that "a traditionalist and an egalitarian respondent might very well both agree on this – The difference is the egalitarian will be pissed and the traditionalist will not."  It really turns on the views imputed to the husband, though, right?  To the extent that egalitarians imagine the chances of the woman having an egalitarian husband, they're going to disagree that problems are almost certain.  This gives the potential for useful information.

**WVS d068 “In your opinion, how important are each of the following traits in a woman? Being a good mother”**:  I think the key here is that the question isn't limited to women who opt into these roles: the stem says "a woman," i.e., any woman.  Women who opt out of these roles are going to be dinged by people who respond that these traits are important, I think.  Can you be a good woman without being a good mother, if you're not a mother at all?  I bet egalitarians are going to respond differently than traditionalists to this.  We'll see, I suppose.

**WVS d069 "a good wife"**  Same.

**WVS d076 “It is acceptable for a man to have more than one wife”**  This question, and the other ones about Islam below, are asked only in Muslim-majority countries.  If they were asked elsewhere, then yes, Islamophobia would totally swamp out anything useful to us.  But as it is, I see them as basically asking whether respondents have a traditional or egalitarian interpretations of Islam with regard to gender roles.  Here, too, the parameter estimates for dispersion will show if I'm mistaken.

**WVS f164, f165, f166** I agree these are sketchier, but in the context of Muslim-majority countries, I think gender egalitarians are more likely to agree than gender traditionalists.  I think we should give them a try.

**WVS f199 (scale of never justifiable to always justifiable) “For a man to beat his wife”** We discussed this one in the meeting, but I'll put it down here for the record anyway.  Of course many traditionalists will recoil from spousal abuse.  But the people who think wife beating is always or almost always justifiable are surely traditionalists.  So this question has, on the one hand, a low difficulty—you don't have to be very egalitarian before you won't stand for domestic abuse—and, on the other, possibly a high dispersion—some even _very_ traditionalist people will give a pretty 'egalitarian' response.  But on average, the responses of more traditionalist people will be different from those of more egalitarian people.  This question should be fine.

Questions I agree should be omitted:

**WVS d070 “In your opinion, how important are each of the following traits in a woman?. Being religious”**  Too likely to be swamped by respondents' views on religion.

**WVS d074 “In your opinion, how important are each of the following traits in a woman?. Protect her family”**  I don't even know what's going on here.  Doesn't everyone think everyone should 'protect' their family?

**WVS d075 “In your opinion, how important are each of the following traits in a woman? Sociability”**  I don't know what's going on here, either.  I don't even know if "real," wife-beating traditionalists _want_ women to be sociable.

**WVS e069_15 “For each one, could you tell me how much confidence you have in them? The women's movement”** Yeah, I think the confidence questions are way too multidimensional.

**WVS f174 “Islam requires that a country with a majority of Muslims be governed by men of Islamic learning”**: I see responses to this question being dominated by the religious/secular aspect rather than the fact that the theocrats are all men rather than women.

Questions I'm not totally sure about:

**WVS e233 “Please tell me for each of the following things how essential you think it is as a characteristic of democracy. Women have the same rights as men”:**  Lindsey says, "This is a question about what defines democracy, not about 'proper' gender roles in society," and I guess I agree.  I could go the other way, though, as I'm pretty sure that egalitarians would on average answer differently from traditionalists.  At bottom, continuing this argument, the question is whether, given that democracy is rule by the people, women should be considered as being fully people.  But again, this is the kind of thoughtful consideration of the question that public opinion scholarship tells us doesn't really happen.  The knee-jerk response to this question is likely often going to be, "no!  Democracy means _everyone_ has the same rights!  _All_ lives matter!" smdh.  Now, that could just mean the dispersion blows up and we should include it anyway.  But, back to the other hand, this is a WVS question.  Any WVS country-year already has a bunch of other questions.  It'll be fine if we drop this one for being too messy.  So, ugh.  Leave it out, or change my mind.

**Comment by byngdeuk** [comment]
Dear Fred and Lindsey, 

I agree with the choice on the questionnaires about what to include and exclude.
I exclude the all questionnaires that Fred points out including WVS e233.

**Comment by fsolt** [comment]
Agreed, @byngdeuk, especially given the countries included in the cdcee: they're coming out from under Soviet communist governments which put considerable emphasis on equality for women to bolster their credibility as egalitarians.  Again, I think DCPO's parameters would likely capture much of what makes this item problematic, but really, there's no need.

**Comment by byngdeuk** [comment]
I reviewed the single-country questionnaires and I find the below issues based on our previous discussion about what kinds of questions should be included and excluded.

1.	First is about “equal opportunity”.
1)	Our Previous discussion
Fred (24 Feb): While I'm looking at these, I don't think we should use LB A505221A, "Employment discrimination by sex: Would you say that in equal conditions of grades, or titles a person is more likely, the same probability, or less likely that would be promoted or accepted in a job?. A woman compared to a man", since it is a positive rather than normative question. In other words, a traditionalist and an egalitarian can very well give the same response to this question—sure, one will be pleased and the other unhappy, but we won't have that information, and so we won't be able to tell them apart. Similar story for ISSP_1985 v34, "Would you say that opportunities for university education <USA: college education> are, in general, better or worse, for women or for men?" There may be others like this, too.
Byung-Deuk: LB A505221A and ISSP_1985 v34 "equal opportunity"
-> I understand the logics. Those questions make respondents to talk about reality, not about opinion. I agree with dropping those kinds of questions.
Lindsey: Employment discrimination and opportunities for education questions:
While these are interesting questions that are certainly related to gender equality, I agree with Fred that we would not be able to discern traditionalist answers from egalitarian answers. This one is more cut-and-dry for me than the question about men being able to handle emotions. I don’t see a way that we could differentiate answers for these questions, so I agree with not using them.

2)	Single-Country questionnaires should be excluded.
 
![image](https://user-images.githubusercontent.com/46687506/85301876-074b5080-b46e-11ea-9b27-1d8d4dbe3206.png)



2.	Questions about the current situation.
I think we have not discussed about the questions asking the current situation. I think it is problematic to include the below questionnaires because they basically ask, “evaluate the current situation.” I do not think those questions can be used to determine whether the respondents are egalitarian or not. I want to discuss about this kind of questions. 
 
![image](https://user-images.githubusercontent.com/46687506/85301909-129e7c00-b46e-11ea-85d9-70ef4fb6cf6e.png)


3.	How to deal with variables without actual questionnaires in the codebook?
 
![image](https://user-images.githubusercontent.com/46687506/85301929-18945d00-b46e-11ea-8f6f-7efa1875be9d.png)



4.	I’m sure that we made a consensus to exclude questions about meetings and association. Thus, I think we should exclude the below questionnaire (I’m sure that there are some other questions. I will double check).
 
![image](https://user-images.githubusercontent.com/46687506/85301953-21852e80-b46e-11ea-9d15-6f05e5ac96e5.png)


5.	Question about Democracy should be excluded based on our recent consensus.
 
![image](https://user-images.githubusercontent.com/46687506/85301971-277b0f80-b46e-11ea-821a-9d465ef776c5.png)


6.	Question about advice.
 
![image](https://user-images.githubusercontent.com/46687506/85301993-2ea21d80-b46e-11ea-9ad0-84b8aa523fb2.png)

I think these are not related to the egalitarian views. Answers highly depend on the personal experience. Thus, those questions should not be included. I want to discuss about this.

**Comment by byngdeuk** [comment]
I updated the changed on the following link: https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/surveys_egr.csv
After we discuss about the "single-country" questions, I will update this too.
(because I can easily make a guess about how busy Lindsey is.)

In addition, after Lindsey update her questions on the above link,
I will start to match the name of variables for the similar (nearly the same) questions.

**Comment by fsolt** [comment]
Re @byngdeuk's comments on single-country questionnaires:

1.1. Good.
1.2. Good.
2.1. Questions asked in only a single country should not be automatically excluded if they are asked repeatedly: the time-series within the country can be useful (cf. https://github.com/fsolt/dcpo_article/blob/bca96e3283552ad2833ae86d5c4cea8fbb998082/paper/dcpo_article.Rmd#L194).  In this list of Australian questions, eXpref/WomPref and eXopp look like they are good questions (X is some digit), have been asked repeatedly, and should be included.  Everything else is problematic for other reasons.  But again, the fact that a question has only been asked in one country is not disqualifying.
2.2. Agreed in principle: we need questions that are normative (asking about R's preferences) not positive (asking about the way things are).  Sure, recognizing a problem is the first step to dealing with it, but seriously these questions are not useful and should be excluded.
3. If the survey is part of a series of surveys that do have codebooks, we can assume with some justification that the question wordings have not changed.  For example, wompref in NSSS_1987 almost certainly tracks WomPref in NSSS_1984, especially given that that wording was barely changed (by adding "and promotions") in the eXpref questions of the later aes_ surveys.
4. Agree that we agreed to exclude these.
5. Yes.
6. I completely agree they should be excluded.

**Comment by fsolt** [comment]
Thanks, too, @byngdeuk, for working on the https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/surveys_egr.csv file.  I was just about to pester you both to do all of your work there so that I can pitch in a bit and we can really get this project rolling.

**Comment by lallemang** [comment]
Hi Fred and Byung-Deuk,

I will see you in our lab meeting in just a few minutes, but below are a couple more questions from some of the cross-national surveys that I want us to double-check before keeping.

**evs_combo d065: _"Men are less able to handle emotions in relationships than women"_**
I'm not sure this will discriminate between egalitarians and traditionalists. Both may agree that men are not as good and handling emotions as women, and the reason why is what will vary (but we won't be able to see that).

**amb_combo vb51: _"Who do you think would be more corrupt as a politician, a man or a woman, or are both the same?"_**
This question just seems odd to me. Would a traditionalist think women are more corrupt? I'm not sure. I know a traditionalist will think women don't belong in politics, so maybe. I would appreciate just hearing your thoughts on this one.

**amb_combo dis12: _"Have you felt discriminated against because of your gender or sex?"_**
This question is asking the respondent for their specific experience, and I'm not sure that helps capture what we're interested in. What does it tell us about egalitarian vs traditionalist views for them to answer this with a yes vs a no? Again, I would just appreciate hearing your thoughts on this question.

**Comment by fsolt** [comment]
Thanks, @lallemang.  As we discussed in this morning's meeting, we agree that these three questions should be excluded (and in fact I think @byngdeuk may have even cut them while he was talking through them).  Just recording our consensus here.

**Comment by byngdeuk** [comment]
Report Before Meeting 

First, I update "single country surveys" on https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/surveys_egr.csv

Second, I found that the below questions are problematic.
![image](https://user-images.githubusercontent.com/46687506/85998661-b76e0b80-b9d0-11ea-8ce2-3887391b8d89.png)

Third, I found that the below questions are problematic .
![image](https://user-images.githubusercontent.com/46687506/85998912-05830f00-b9d1-11ea-9dc4-d6e6588bfb7f.png)

Fourth, I don't know how to handle the below questions. There are no previous or recent questions about this. 
![image](https://user-images.githubusercontent.com/46687506/85999076-37947100-b9d1-11ea-9f26-8aa2db7539a7.png)

Lastly, I think the below "normative" questions can be included (I did not update the below yet).
![image](https://user-images.githubusercontent.com/46687506/85999177-58f55d00-b9d1-11ea-848f-e52040a03d51.png)

See you soon!!

**Comment by fsolt** [comment]
1.  Thanks!
2.  a.  If these questions are asked only once, they are definitely not useful, no matter what the q actually says.  b.  Yeah, I have to agree that I don't know how to rank all of these responses on our dimension.  They're out.
3.  Right.  They're out.
4.  The first and third appear to me to be an additive index, so we're not interested.  If there's no womcfam in 1984, then given the absence of a codebook for 1987 we'll have to give up on it.
5. Yes, I pointed out these are good questions [up here](https://github.com/fsolt/dcpo_gender_roles/issues/1#issuecomment-647591625).  They should go in.

See you tomorrow!

**Comment by byngdeuk** [comment]
I continuously think that our meeting is on Monday haha. 

I updated AES surveys on the following link:  https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/surveys_egr.csv

I definitely need Lindsey to double check the code.
In addition, Lindsey and I can work on "matching" variable name task after Lindsey upload her portion on that link.
Please let me know if you have any problem @lallemang .
See you tomorrow!

**Comment by byngdeuk** [comment]
I finished matching up the variable names on the following link:  https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/surveys_egr.csv.

I drop some variables which are included even though we made consensus to exclude.
I modify some mistakes.

I hope Lindsey can review my work!

If there are anything boosting our work, please let me know!

**Comment by fsolt** [comment]
Thanks, Byung-Deuk!  If you'd like to push forward, check the [open issues](https://github.com/fsolt/dcpo_gender_roles/issues), choose one to start, assign yourself (on the right, towards the top), and go to it.

**Comment by byngdeuk** [comment]
Dear @fsolt and @lallemang !
I updated both of the following links: https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/gender_role_items.csv and https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/surveys_egr.csv.
There are lots of various and useful questionnaires for us. 

Lindsey! Please double check my works.

I will add some more questions after I get a confirmation about  survey names (I made comments on the other sites). 

There is one question that I do not know how to handle with.


pewrel2009_afr, Q85G.
The questionnaire is "whether you personally believe that it is morally acceptable, morally wrong, or is it not a moral issue? Polygamy – having more than one wife ".
Respondents can answer to this by "1 Morally acceptable / 2 Morally wrong / 3 Not a moral issue / 4 Depends on situation".

I wonder how to order the answers. 
I first think "c(1, 3, 4, 2)".
I want to listen your opinions.

Please let me know if there are additional works needed to boost our work!
(I'm starving for the publication after a few rejections :)).

Thanks!

**Comment by lallemang** [comment]
**Below are my thoughts on some of the questions added from the Arab Barometer surveys:**

I think the following questions ask the respondent for their interpretation of Islam, rather than their opinion on gender roles. Although, it’s plausible that more progressive Muslims interpret religious doctrine through a more egalitarian lens; in which case, answers to these questions might still reveal something about opinions on gender roles:

- **arabb1,islamclass4,q5044,c(4:1):** _“Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that … It is acceptable in Islam for male and female university students to attend classes together.”_

- **arabb2,permitsec4,q60112,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … The first wife’s consent is a prerequisite for permitting a man to marry a second woman.”_

- **arabb2,rejmarry4,q60113,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A woman can reject a marriage that her family chose for her without her consent.”_

- **arabb3,workout4a,q6012,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement or disagreement with each statement … A married woman can work outside the home.”_

- **arabb4,islamclass4,q6074,c(4:1):** _“Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that is presented … It is acceptable in Islam for male and female university students to attend classes together.”_

I think the following questions contain two statements within a single question, making it difficult to know which statement the respondent is evaluating. They also ask the respondent for their interpretation of Islam, rather than their opinion on gender roles. We might not be worried about either of these points, but I want to point them out just in case:

- **arabb1,hijab4a,q5046, c(4:1):** _“Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that … In Islam a woman should dress modestly, but Islam does not require that she wear a hijab.”_ 

- **arabb2,hijab4b,q6076,c(4:1):** _“The opinions of Islamic jurists and religious scholars differ with regard to their interpretations of certain issues in Islam. I want to ask to what extent you agree or disagree with some of these issues … Women should wear modest clothes without needing to wear hijab.”_ 

- **arabb3,hijab4b,q6076,c(4:1):** _“The opinions of Islamic jurists and religious scholars differ with regard to their interpretations of certain issues in Islam. I want to ask to what extent you agree or disagree with some of these issues … Women should wear modest clothes without needing to wear a hijab.”_

- **arabb4,hijab4a,q6076,c(4:1):** _“Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that is presented … A woman should dress modestly, but Islam does not require that she wear a hijab.”_

I think that the following questions ask the respondent to evaluate the current status of women in society, not provide their own opinion on gender roles. Therefore, answers to these questions won’t reveal information about preferences/attitudes/opinions, which is what we’re actually interested in:

- **arabb2,preprime4,q60101,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A woman can become the prime minister or president of a Muslim state.”_ 

- **arabb2,workout4a,q60102,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A married woman can work outside the home.”_ 

- **arabb2,travel4,q60107,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … It is permissible for a woman to travel abroad by herself.”_ 

- **arabb2,judicial4,q60110,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … Women can assume judicial positions.”_ 

- **arabb2,wminister4,q60111,c(4:1):** _“I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … Women can become ministers.”_

**Comment by byngdeuk** [comment]
I think we already made a consensus on the questionnaires Lindsey mentioned. 

First, we made a consensus to include some questions related to religion.
fsolt commented that "WVS d076 “It is acceptable for a man to have more than one wife” This question, and the other ones about Islam below, are asked only in Muslim-majority countries. If they were asked elsewhere, then yes, Islamophobia would totally swamp out anything useful to us. But as it is, I see them as basically asking whether respondents have a traditional or egalitarian interpretations of Islam with regard to gender roles. Here, too, the parameter estimates for dispersion will show if I'm mistaken.
WVS f164, f165, f166 I agree these are sketchier, but in the context of Muslim-majority countries, I think gender egalitarians are more likely to agree than gender traditionalists. I think we should give them a try."

**Thus, I think we should include the below questions.** Moreover, the below five questions include “can”, “permissible”, or “can assume” and I think they ask respondent’s normative views.
•	arabb2,preprime4,q60101,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A woman can become the prime minister or president of a Muslim state.”
•	arabb2,workout4a,q60102,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A married woman can work outside the home.”
•	arabb2,travel4,q60107,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … It is permissible for a woman to travel abroad by herself.”
•	arabb2,judicial4,q60110,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … Women can assume judicial positions.”
•	arabb2,wminister4,q60111,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … Women can become ministers.”


About “hijab” questions,
Lindsey mentioned that “WVS f164 “Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, would you please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that is presented? Islam requires woman to dress modestly but does not require cover face with veil”: Again, I don’t think we’re trying to measure views toward traditional religious practices.”
Fred mentioned that “WVS f164, f165, f166 I agree these are sketchier, but in the context of Muslim-majority countries, I think gender egalitarians are more likely to agree than gender traditionalists. I think we should give them a try.”

**Considering the fact that the questionnaires are actually the same, we should include the below questions.**
•	arabb1,hijab4a,q5046, c(4:1): “Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that … In Islam a woman should dress modestly, but Islam does not require that she wear a hijab.”
•	arabb2,hijab4b,q6076,c(4:1): “The opinions of Islamic jurists and religious scholars differ with regard to their interpretations of certain issues in Islam. I want to ask to what extent you agree or disagree with some of these issues … Women should wear modest clothes without needing to wear hijab.”
•	arabb3,hijab4b,q6076,c(4:1): “The opinions of Islamic jurists and religious scholars differ with regard to their interpretations of certain issues in Islam. I want to ask to what extent you agree or disagree with some of these issues … Women should wear modest clothes without needing to wear a hijab.”
•	arabb4,hijab4a,q6076,c(4:1): “Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that is presented … A woman should dress modestly, but Islam does not require that she wear a hijab.”




**In the same vein, I think we should include the below questionnaires. I think gender egalitarians are more likely to agree to the below questionnaires than gender traditionalists.**
•	arabb1,islamclass4,q5044,c(4:1): “Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that … It is acceptable in Islam for male and female university students to attend classes together.”
•	arabb2,permitsec4,q60112,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … The first wife’s consent is a prerequisite for permitting a man to marry a second woman.”
•	arabb2,rejmarry4,q60113,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A woman can reject a marriage that her family chose for her without her consent.”
•	arabb3,workout4a,q6012,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement or disagreement with each statement … A married woman can work outside the home.”
•	arabb4,islamclass4,q6074,c(4:1): “Today as in the past, Muslim scholars and jurists sometimes disagree about the proper interpretation of Islam in response to present-day issues. For each of the statements listed below, please indicate whether you agree strongly, agree, disagree, or disagree strongly with the interpretation of Islam that is presented … It is acceptable in Islam for male and female university students to attend classes together.”

If we change our decisions on the above questions, we should drop the similar or the same questionnaires already included in the following link  https://github.com/fsolt/dcpo_gender_roles/blob/master/data-raw/surveys_egr.csv.

**Comment by fsolt** [comment]
Okay, first, I'll note that if Lindsey is raising a concern, then it's not correct to respond, Byung-Deuk, that we've reached consensus—she clearly does not agree.

From my perspective, I think Lindsey's first two groups of items are likely going to be fine.  If you look at the religion items in these countries, respondents overwhelmingly identify as Muslim and consider their religion important to their lives.  In such contexts, I contend, whether a gender item is prefaced with something about interpretations of Islam really doesn't matter much: in these societies, questions of gender will _always_ be viewed through the lens of an interpretation of Islam.  In any event, we can treat whether these items are suitable as an empirical question, and check their dispersion parameters in our initial runs.  I think this is all that Lindsey is advocating, and it's a point well taken.  In fact, Lindsey, I'll ask you to open a new issue to keep a list of items that we will need to check up on later.  The WVS questions that Byung-Deuk quotes me on should go there, too, of course.

Lindsey's third group of items is more difficult.  The “can”, “permissible”, or “can assume” language is at best ambiguous.  I can see Byung-Deuk's point that they invite a normative response, but really, I view these as pretty positive.  And the fact of the matter is, given that we're going to tentatively include the first two sets of items, which are definitely better, we don't need these items.  Can we agree to drop them, Byung-Deuk?

**Comment by byngdeuk** [comment]
I'm really sorry that I respond in such way. I will correct my way to respond.
If my previous way to respond made you feel bad Lindsey, please forgive me. It was not my intention.
I agree to drop the below question because now I understand why those questions ambiguous. 

arabb2,preprime4,q60101,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A woman can become the prime minister or president of a Muslim state.”
arabb2,workout4a,q60102,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … A married woman can work outside the home.”
arabb2,travel4,q60107,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … It is permissible for a woman to travel abroad by herself.”
arabb2,judicial4,q60110,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … Women can assume judicial positions.”
arabb2,wminister4,q60111,c(4:1): “I will read a set of statements that relate to the status of women in our society to you in order to gauge the extent of your agreement/disagreement with each statement … Women can become ministers.”

Again, I was totally my bad if my previous comment made Lindsey uncomfortable and feel offended.
I will change my way to respond.

**Comment by byngdeuk** [comment]
I write this in order to track our work :)

In the last meeting,  we first talked about religion related questionnaires and we decide to include those questions. We will keep tracking those questions. Second, we talked about names of the surveys in DCPO project and the actual survey names for pewrel2014_us and pewrel2016_us. 

For now, I checked the pewrel2014_us and pewrel2016_us. However, there are no questions related to gender roles.

I'm looking forward to the next meeting and please let me know if there are anything that I can do for our project before the next meeting!

**Comment by byngdeuk** [comment]
I get the below questionnaires from the additional surveys that Fred posted yesterday.

![image](https://user-images.githubusercontent.com/46687506/88659122-24d98e80-d09a-11ea-849f-aab8a1be6f83.png)

**Comment by byngdeuk** [comment]
I reviewed the surveys that Fred listed yesterday. I have the below questions from the surveys.

**Full Lists**
jgss | Q4WNMGA | View on Connection between Women Happiness and Marriage
jgss | Q4WNJB2L | View on Women Independence by Having Job
jgss | Q4WWJBIA | View on Wife Working
jgss | Q4WWHPHH | View on Role of Wife to Help Husband
jgss | Q4WWDVY | View on Desirability of Divorce for Children Wife
jgss | RR6ACCT | Household Budget Management
jgss | OP4NAME | View on Change of Surname at Marriage
kgss | FEJOBIND | Agree or disagree: having a job is a best way for a woman to be   independent
kgss | FAMSUFFR | Agree or disagree: family life suffers when the woman takes a full-time   job
kgss | TWOINCS | Agree or disagree: both man and woman should contribute to the household   income
kgss | HUBBYWRK | Agree or disagree: a man's job is to earn money; a woman's job is to look   after
kgss | UAFOPFF | Agree or disagree on financial support: unmarried adult woman for her own   parent
kgss | MAFOPFF | Agree or disagree on financial support: married adult woman for her own   parent(s)
kgss | MAFLPFF | Agree or disagree on financial support: married adult woman for her   parent(s)
kgss | FEWRKSUP | Agree or disagree: most women have to work to support the family
kgss | KIDSUFFR | Opinion on women's employment: a pre-school child is likely to   suffer if his or
kgss | HOMEKID | Opinion on women's employment: a job is all right, but what   most women really wa
kgss | SEXROLE4 | Opinions on gender roles: During economic recession, it is all   right for women t
kgss | HBBYWK08 | Agree or disagree: Husband's job is to earn money; wife's job   is to look after t
kgss | HOUSEWRK | Agree or disagree: being a housewife is just as fulfilling as working for   pay
kgss | SEXROLE1 | Opinions on gender roles: It is more important for a wife to   help her husband’
kgss | SEXROLE2 | Opinions on gender roles: A husband’s job is to earn money; a   wife’s job is
kgss | SEXRL108 | Opinions on gender roles: It is more important for a wife to   help her husband’
usgss | FEPRIEST | It would be a good thing if women were allowed to be ordained as priests.
usgss | FECLERGY | Do you favor or oppose women as pastors, ministers, priests, or rabbis in   your own faith or denomination?
usgss | FEHOME | Do you agree or disagree with this statement? Women should take care of   running their homes and leave running the country up to men.
usgss | FEPOL | Most men are better suited emotionally for politics than are most women.
usgss | FEPOLY | Would you say that most men are better suited emotionally for politics   than are most women, that men and women are equally suited, or that women are   better suited than men in this area?
usgss | ERATELL | The Equal Rights Amendment will amend the United States Constitution to   provide equal rights for men and women. Do you strongly favor, somewhat   favor, somewhat oppose, or strongly oppose this amendment?
usgss | FEFIGHT | As far as you know, are women now assigned to jobs in the armed forces   that would expose them to combat, or are women not assigned to such jobs?
usgss | FEDIRTY | As far as you know, are women in the armed forces now assigned to dirty   jobs like repairing trucks or other heavy equipment, or are women not   assigned to such jobs?
usgss | FEBRASS | As far as you know, are women in the armed forces now assigned to jobs   where they have command over men, or are women not assigned to such jobs?
usgss | FELESS1 | Men work harder on the job than women do. How important do you think this   reason is for explaining why women earn less?
usgss | FEKIDS1 | Women are biologically better-suited to care for children. How important   do you think this reason is for explaining why women are more likely to take   care of children?
usgss | FEKIDS5 | t is God's will that women care for children. How important do you think   this reason is for explaining why women are more likely to take care of   children?
usgss | HOMEKID | A job is alright, but what most women really want is a home and children.
usgss | WRKNOKID | Do you think that women should work outside the home full-time, part-time   or not at all under these circumstances: After marrying and before there are   children.
usgss | FEWRKSUP | Do you agree or disagree . . . Most women have to work these days to   support their families.
usgss | FEWKNOKD | Do you think that women should work outside the home full-time, part-time   or not at all, when a couple has not yet had a child?
usgss | FEWORK | Do you approve or disapprove of a married woman earning money in business   or industry if she has a husband capable of supporting her?
usgss | FEPRES | If your party nominated a woman for President, would you vote for her if   she were qualified for the job?
usgss | ABANY | Please tell me whether or not you think it should be possible   for a pregnant woman to obtain a legal abortion if. . . The woman wants it   for any reason?
usgss | ABSPNO | If a woman has decided to have an abortion but her husband is   against it, do you think she should go ahead and have it, or not?
usgss | FEFAM | It is much better for everyone involved if the man is the achiever   outside the home and the woman takes care of the home and family.
usgss | FEWORKIF | If there is a limited number   of jobs, do you approve or disapprove of a married woman holding a job in   business or industry when her     husband is able to support her?
usgss | FIGHTAIR | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A jet fighter   pilot
usgss | MECHANIC | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A truck mechanic
usgss | NURSE | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A nurse in a   combat zone
usgss | TYPIST | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A typist in the   Pentagon in Washington
usgss | BRASS | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. Commander of a   large military base
usgss | FIGHTLND | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. Soldier in   hand-to-hand combat
usgss | TRANSAIR | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A jet transport   pilot
usgss | GUNNER | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. Air defense   missile gunner in the United States
usgss | FIGHTSEA | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A crew member on a   combat ship
usgss | TRADMOD | which type of relationship would you prefer?
usgss | OPSEX | Being born a man or a woman -- how important is that for getting ahead in   life?
usgss | FAMSUFFR | All in all, family life suffers when the woman has a full-time job.
usgss | HAPIFWRK | A woman and her family will all be happier if she goes out to work.
usgss | FEJOBIND | Having a job is the best way for a woman to be an independent person.
usgss | HUBBYWK1 | A man's job is to earn money; a woman's job is to look after the home and   family.
usgss | MRMOM | It is not good if the man stays at home and cares for the children and   the woman goes out to work.
usgss | ABCHOOSE | A pregnant woman should be able to obtain a legal abortion for   any reason whatsoever, if she chooses not to have the baby.

**The below questions do not have full labels about the actual questionnaires. The link for the codebook about "kgss" does not work.** 
kgss | KIDSUFFR | Opinion on women's employment:   a pre-school child is likely to suffer if his or
kgss | HOMEKID | Opinion on women's employment: a job is all right, but what   most women really wa
kgss | SEXROLE4 | Opinions on gender roles: During economic recession, it is all   right for women t
kgss | HBBYWK08 | Agree or disagree: Husband's job is to earn money; wife's job   is to look after t
kgss | SEXROLE1 | Opinions on gender roles: It   is more important for a wife to help her husband’
kgss | SEXROLE2 | Opinions on gender roles: A husband’s job is to earn money; a   wife’s job is
kgss | SEXRL108 | Opinions on gender roles: It is more important for a wife to   help her husband’

**I think we can include the below questions. I want to listen your opinions.**
**Job Related**
usgss | FIGHTAIR | Please tell me whether you  think a woman should or should not be assigned to each job, assuming she is   trained to do it. A jet fighter pilot
usgss | MECHANIC | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A truck mechanic
usgss | NURSE | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A nurse in a   combat zone
usgss | TYPIST | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A typist in the   Pentagon in Washington
usgss | BRASS | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. Commander of a   large military base
usgss | FIGHTLND | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. Soldier in   hand-to-hand combat
usgss | TRANSAIR | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A jet transport  pilot
usgss | GUNNER | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. Air defense  missile gunner in the United States
usgss | FIGHTSEA | Please tell me whether you think a woman should or should not   be assigned to each job, assuming she is trained to do it. A crew member on a   combat ship

**Abortion related**
usgss | ABANY | Please tell me whether or not   you think it should be possible for a pregnant woman to obtain a legal   abortion if. . . The woman wants it for any reason?
usgss | ABSPNO | If a woman has decided to have an abortion but her husband is   against it, do you think she should go ahead and have it, or not?
usgss | ABCHOOSE | A pregnant woman should be   able to obtain a legal abortion for any reason whatsoever, if she chooses not   to have the baby.

**Additional One** 
usgss | TRADMOD | which type of relationship   would you prefer?
1 A relationship   where the man has the main responsibility for providing the household income   and the woman has the main responsibility for taking care of the home and   family / 2 A relationship where the man and woman equally share   responsibility for providing the household income and taking care of the home and family.

I try to include all the questionnaires related to gender role. However, it will be great Lindsey to review the codebooks of the surveys and catch my possible mistake.. 

See you soon!

**Comment by fsolt** [comment]
### Update

Lots of the items we had questions about ended up being dropped for not being asked often enough: countries that don't have two years are excluded, and then items that don't have at least five country-years are excluded.

On a few of the others that Lindsey raised concerns about, the dispersion parameters in the preliminary run proved her right (and me wrong):

**wvs d076** "It is acceptable for a man to have more than one wife" 
**wvs d067** "In your opinion, how important are each of the following traits in a woman? Wearing a veil in public places"
**pew2010 q108a** "Do you favor or oppose making the following the law in (survey country)?  Segregation of men and women in the workplace"
**arabb2 q60115** "The law should not allow a man to marry more than one woman."

I'm going to exclude them.  None of these items brought us any new country-years, so no worries on that front.  

Although this completely wipes out polygamy as a topic--as Lindsey anticipated, it appears it is too cultural to help us--the other veil/hijab questions (Pew's "Women should have the right to decide if they wear a veil" and the Arab Barometer's "A woman should dress modestly, but Islam does not require that she wear a hijab" and "Women should wear modest clothes without needing to wear a hijab.") were definitely useful.  So ¯\_(ツ)_/¯

**Comment by fsolt** [comment]
Other items I'm excluding on the grounds that in the countries surveyed these pick up religion and/or religiosity more than gender roles:

**pew2009_afr** "Please tell me whether the FIRST statement or the SECOND statement comes closer to your own views ?even if neither is exactly right. Women's religious ledaership roles: 1 Women should be allowed to serve in religious leadership roles, such as pastor, priest or imam / 2 Only men should be able to serve in religious leadership roles, such as pastor, priest or imam"
**pew2014_latam** "Do you think Protestant/Evangelical churches should allow women to become pastors, or do you not think so?"
**pew2014_latam** "And do you think the Catholic Church should or should not (INSERT NEXT ITEM)? Allow women to become priests"
**pew2015_ee** "Do you favor or oppose the Orthodox Church’s current position of: Forbidding women from becoming priests"
**pew2015_ee** "Do you think the Catholic Church should or should not: Allow women to become priests?"

**Comment by fsolt** [comment]
This one is positive, not normative, so I'm dropping it:

**pew 2010 q47** "Men get more opportunities than women for jobs that pay well, even when women are as qualified as men for the job. Do you completely agree, mostly agree, mostly disagree or completely disagree?"

**Comment by byngdeuk** [comment]
I agree with the above dropping variable things! Thanks for reviewing our work!!

**Comment by fsolt** [comment]
So dropped our country-year-item observations to 5588 from 5864, reshuffled the top 12 countries a bit, with Mexico dropping out and Slovenia added.  That's a shame--it makes sense given that we lost three items from lb and two from pew2014_latam, of course--but at least we have more regional diversity in Fig 3a.  See https://github.com/fsolt/dcpo_gender_roles/blob/4453c9b997e48557482a044351c457cb5c26324f/paper/dcpo_gender_roles.pdf
